src/schema/me/index.js

```diff
@@ -115,9 +113,7 @@ export default {
       "followsAndSaves",
     ]
     if (queriedForFieldsOtherThanBlacklisted(fieldNodes, blacklistedFields)) {
-      return gravity
-        .with(accessToken)("me")
-        .catch(() => null)
+      return meLoader().catch(() => null)
```

Comment:
Any idea what errors this catches and why returning `null` is preferable? Because I’m not sure we’d want clients to not get these errors.

Reply:
I'm assuming return `null` if you're not logged in, essentially?

Reply:
I think this may actually be leading to state not being handled well in the app, because it expects errors to signal a state like `401`, but we can follow-up with that later.